public/catch-js-errors.js

```diff
@@ -90,7 +90,7 @@
 
 	if ( isLocalStorageNameSupported() ) {
 		// Randomly assign 1% of users to log errors
-		if ( localStorage.getItem( 'log-errors' ) === undefined ) {
+		if ( ! localStorage.getItem( 'log-errors' ) ) {
 			if ( Math.random() <= 0.01 ) {
```

Comment:
I checked the old IRC-based implementation and we were throttling to 1% of users there too. Without diving more deeply into it I'd probably start there and iterate if we find it's not enough or that it's too much.